Library/Formula/node.rb

```diff
@@ -1,8 +1,9 @@
 class Node < Formula
   desc "Platform built on the V8 JavaScript runtime to build network applications"
   homepage "https://nodejs.org/"
-  url "https://nodejs.org/dist/v5.7.0/node-v5.7.0.tar.gz"
-  sha256 "2338b46a2f45fbb747089c66931f62555f25a5928511d3a43bbb3a39dcded2d8"
+  url "https://nodejs.org/dist/v5.7.1/node-v5.7.1.tar.gz"
+  mirror "https://github.com/nodejs/node/archive/v5.7.1.tar.gz"
+  sha256 "dc6ad2f7c5ac0b87db5b20731c42a1c4fc22edb6a5da22ca93fe9ffd3eac95ce"
```

Comment:
Fair enough. At the time, I was getting periodic 404s from nodejs.org. Things seem better now.

Thanks